Gemfile.lock

```diff
@@ -752,4 +752,4 @@ DEPENDENCIES
   webmock (~> 3.23)
 
 BUNDLED WITH
-   2.5.7
+   2.5.9
```

Comment:
This was updated automatically. Should I revert this change?

Reply:
No, it's fine.